src/vscode-bicep-ui/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@vscode-bicep/ui",
+  "private": true,
+  "packageManager": "npm@10.8.1",
+  "workspaces": [
+    "apps/*",
+    "packages/*"
+  ],
+  "scripts": {
+    "build": "turbo build",
```

Comment:
1. We should validate this builds in our GH Actions pipeline
2. We should connect this to the dependabot config, so that security fixes are picked up

Reply:
Created an issue to track the work: https://github.com/Azure/bicep/issues/14589.